### PR TITLE
Feature/48 Wrote FreeParkingSquare Class

### PR DESCRIPTION
**Pull request by Dcbaas**
This class was so trivial that we forgot to write up an issue for it.

### CODE REVIEW

File: Model/src/BoardPackage/FreeParkingSquare.java [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
-/**
- * Created by Dylan Kernohan on 2/10/18.
- */
-public class FreeParkingSquare {
+/*********************************************************************
+ * The FreeParkingSquare is a board square on the Monopoly game board
+ * where when you land on it nothing happens. This class exist however
+ * for the possiblity of adding functionality for this board square
+ * later.
```
Comment (quirogas) [inline Model/src/BoardPackage/FreeParkingSquare.java]: Add the version and the author to the class description.

Comment (kernohad) [reply Model/src/BoardPackage/FreeParkingSquare.java]: Agreed

Comment (dustinecker) [reply Model/src/BoardPackage/FreeParkingSquare.java]: ^

Comment (dustinecker) [reply Model/src/BoardPackage/FreeParkingSquare.java]: ^

**Pull request merged**